modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,272 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-classic-aws-rhcos-ami_{context}"]
+= {product-title} AMIs 
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {product-title} nodes.
+
+[id="rosa-classic-aws-rhcos-ami-415_{context}"]
+== Version 4.15 AMIs
+
```

Comment:
@michaelryanmcneill, @iamkirkbater, and @yuwang-RH same clarifying questions as the 4.14 section

Reply:
I'm confused - where is the ROSA CLI getting the list of AMIs from?

I'd trust that list over my own, I gathered mine directly from AWS, so I'd trust the ROSA CLI since those are contained within `overrides` - I guess I just don't understand why there's so many overrides. 

Do we know why there are published ROSA AMIs on AWS and where the overridden ones come into play? Like most things with ROSA this hidden complexity just doesn't make sense without the historical context :(

Should this rosa command be the source of truth here?

Reply:
@michaelryanmcneill and @kmcolli any thoughts on this? I'm really not sure of the best approach anymore.

Reply:
We're going to need to understand what the right answer to this is, but I don't know what it is.